src/arrayMethodSort.js

```diff
@@ -4,8 +4,28 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i; j < this.length; j++) {
+        if (compareFunction) {
+          if (compareFunction(this[i], this[j]) > 0) {
```

Comment:
It's good to provide a default compare function in case `compareFunction` is not provided. This will ensure the method behaves correctly according to the ECMAScript specification for `Array.prototype.sort`.